lib/model.js

```diff
@@ -958,6 +962,10 @@ Connection.getFieldNames = function(authentication) {
   return AUTHENTICATION_TO_FIELD_NAMES[authentication];
 };
 
+Connection.isAtlas = function(str) {
+  return str.indexOf('.mongodb.net') > -1;
```

Comment:
for the UI this is probably sufficient (i.e. no attack vectors, all it does is set the ssl field to "UNVALIDATED". More generally though, I could imagine where we use the `isAtlas()` function for other security related features and then there's the danger of someone maliciously pretending to be an atlas instance by naming their instance `mongodb.net.myserver.com`. 

Can we do something cheap to make this a bit safer? Like test for `.mongodb.net[:/]` or something? Or by parsing into mongodb-connection-model and then only checking if the `hostname` field ends in `mongodb.net` ?